.github/workflows/cd-bump-homebrew.yaml

```diff
@@ -1,11 +1,6 @@
 name: Bump Dolt on Homebrew
 
 on:
-  workflow_dispatch:
-    inputs:
-      version:
-        description: 'SemVer format release tag, i.e. 0.24.5'
-        required: true
```

Comment:
we want to keep this, as sometimes we need to run the workflow manual after a release has already run.

Reply:
you can dedupe the code by doing stuff like:
```
tag-name: ${{format('refs/tags/v{0}', github.event.client_payload.version || github.event.inputs.version)}}
```

where `github.event.client_payload.version` is from the `workflow_dispatch` event and `github.event.inputs.version` is from the `repository_dispatch` event.